wouso/games/challenge/forms.py

```diff
@@ -34,13 +34,14 @@ def check_self_boxes(self):
             for j in range(len(field.field.choices)):
                 choice = field.field.choices[j]
                 if str(choice[0]) in checked_boxes:
-                    self.visible_fields()[i].field.choices[j] = (choice[0], choice[1], True)
+                    self.visible_fields()[i].field.choices[j] = (choice[0],
+                                                                 choice[1],
+                                                                 True)
 
     def get_results_in_order(self, results):
         """ Assign question feedback with according question """
         v = []
         for field in self.visible_fields():
             id = int(field.html_name[len('answer_'):])
-            v.append( results[id] )
-        return v
-
+            v.append(results[id])
+        return v
```

Comment:
"forms.py:46:1: W391 blank line at end of file"
i just did what the issues was saying.